### ISSUE TITLE
Can't replicate results

### ISSUE THREAD

**Issue by Seraphid** [sgrvinod/a-PyTorch-Tutorial-to-Image-Captioning#120]
Hi, I'm trying to replicate the results reported in the README. So far, without changing anything except the paths, I got 14.25 BLEU in the 12th epoch instead of the 23.25 that you get in the README. Is there something I have to change from the code to reach that part? I have checked and it used Adam with the recommended learning rate.

Thanks in advance

**Comment by Seraphid** [comment]
The problem is with the imgread. If you import it from another package, it will load the images in the scale [0,1]. Therefore, you need to multiply them by 255 or store them as a float in the HDF5 file.

**Comment by ragvri** [comment]
Where is imgread being used? I am facing the exact same issue
